### DISCUSSION TITLE
Dev Chat

### DISCUSSION THREAD

**Discussion by alensiljak** [orgzly-revived/orgzly-android-revived#66] [Ideas]
For a quick dev team communication, I've created a room at gitter.im:

https://app.gitter.im/#/room/#orgzly-revived-chat:gitter.im

We can temporary (or not) use this until we establish a better solution. I haven't used gitter much but this might work. The room is currently public.

**Comment by cbkerr** [comment]
There's also a Matrix chat room, #13

**Reply by alensiljak** [reply]
Cool, didn't know about that. I need something for quick consultations instead of going through Issues or Discussions.
I.e. trying to resolve the build pipeline currently.

**Reply by cbkerr** [reply]
I don't have experience with matrix or gitter but I am fine with any open source and open platform :)

**Reply by alensiljak** [reply]
Neither do I. Apparently there's some interaction between Matrix and Gitter, too.
